pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -234,6 +234,13 @@
             "#import": [ "rhel", "linux-x64" ]
         },
 
+        "rhel.6": {
```

Comment:
That depends on the compatibility between these releases.  I would imagine RHEL has strong compat guarantees like Windows so I would lean towards making rhel.7 import rhel.6.  Ultimately we would want RedHat to make that call though.